### DISCUSSION TITLE
Welcome to nim-deploy Discussions!

### DISCUSSION THREAD

**Discussion by supertetelman** [NVIDIA/nim-deploy#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aliyevaladddin** [comment]
Hi NVIDIA team! 👋

I'm Aladdin, building AladdinAI — an open-source BYOI (Bring Your Own 
Infrastructure) AI Agent platform with native NVIDIA NIM integration.

GitHub: https://github.com/aliyevaladddin/AladdinAI

What we've built with NIM:
- Native NIM provider with proper tool_choice=auto for autonomous agent execution
- Disabled parallel_tool_calls for full NIM compatibility  
- Multi-agent orchestration powered by NIM models
- Code review agent running on NIM
- Self-hosted — all data stays on your infrastructure

The platform also includes browser-based terminal, IMAP email integration 
(agents read & reply to emails autonomously), WhatsApp/Telegram control 
of agents, and built-in CRM — all running on your own GPU stack via NIM.

Would love to connect with the NIM team and explore deeper collaboration. 
Happy to share a demo!
